facades/PC/src/main/java/org/terasology/engine/Terasology.java

```diff
@@ -150,13 +153,32 @@ public static void main(String[] args) {
             if (isHeadless) {
                 engine.subscribeToStateChange(new HeadlessStateChangeListener(engine));
                 engine.run(new StateHeadlessSetup());
+            } else if (loadLastGame) {
+                engine.initialize(); //initialize the managers first
+                engine.getFromEngineContext(ThreadManager.class).submitTask("loadGame", () -> {
+                    GameManifest gameManifest = getLatestGameManifest();
+                    if (gameManifest != null) {
+                        engine.changeState(new StateLoading(gameManifest, NetworkMode.NONE));
+                    }
+                });
             } else {
-                if (loadLastGame) {
-                    engine.initialize(); //initialize the managers first
-                    engine.getFromEngineContext(ThreadManager.class).submitTask("loadGame", () -> {
+                if (createLastGame) {
+                    engine.initialize();
+                    engine.getFromEngineContext(ThreadManager.class).submitTask("createLastGame", () -> {
                         GameManifest gameManifest = getLatestGameManifest();
                         if (gameManifest != null) {
-                            engine.changeState(new StateLoading(gameManifest, NetworkMode.NONE));
+                            String title = gameManifest.getTitle();
+                            GameManifest newlyCreatedGM = gameManifest;
+                            if (!title.startsWith("New Created")) { //if first time run
+                                newlyCreatedGM.setTitle("New Created " + title + " 1");
+                            } else { //if not first time run
+                                String newTitle = title.substring(0, title.length() - 2);
+                                //-2 to skip the last number added
+                                char fileNumber = title.charAt(title.length() - 1);
+                                fileNumber++;
+                                newlyCreatedGM.setTitle(newTitle + " " + fileNumber);
+                            }
+                            engine.changeState(new StateLoading(newlyCreatedGM, NetworkMode.NONE));
```

Comment:
I was able to get it to increment instead of a random number, I couldn't get parentheses or brackets around the number, so I just added it to the end. This code shouldn't run into any problems, unless someone creates a world with a name that starts with "New Created" and end with a number. Which I doubt will happen.